cli/internal/runcache/runcache.go

```diff
@@ -162,8 +170,7 @@ func (tc *TaskCache) RestoreOutputs(ctx context.Context, prefixedUI *cli.Prefixe
 	default:
 		// NoLogs, do not output anything
 	}
-
-	return true, nil
+	return cacheStatus, nil
```

Comment:
Presumably either local or remote is true at this point?

Reply:
yeah it shuold just be one of them. I'm not loving this cacheStatus struct all around, but I didn't want to change that here, just passing through the information we have